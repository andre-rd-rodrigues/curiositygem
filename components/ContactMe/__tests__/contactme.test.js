import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import ContactMe from "../ContactMe";

describe("Contact me", () => {
  it("renders component unchanged", () => {
    const { container } = render(<ContactMe />);

    expect(container).toMatchSnapshot();
  });
});
