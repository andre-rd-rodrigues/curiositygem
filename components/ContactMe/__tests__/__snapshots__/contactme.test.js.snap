// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact me renders component unchanged 1`] = `
<div>
  <p>
    If you have any futher questions or doubts about this topic,
     
    <a
      href="mailto:andrerodrigo.web@gmail.com"
    >
      send me a message
    </a>
     and I'll be happy to help!
  </p>
</div>
`;
